kalite/coachreports/static/css/coachreports/tabular_view.css

```diff
@@ -80,7 +80,7 @@ th.headrowuser {
 	background-color: #eee;
 }
 
-#displaygrid td, #displaygrid th {
+#displaygrid td.status, #displaygrid th {
```

Comment:
Make this more specific to prevent it overriding cells in the details panel table.